sematic/config/settings.py

```diff
@@ -126,6 +130,7 @@ def get_active_settings() -> ProfileSettings:
 
         profile_settings = settings.profiles[_DEFAULT_PROFILE]
 
+        _apply_default_scopes(profile_settings.scopes)
```

Comment:
This is to make sure that `sematic.plugins.storage.local_storage` is imported when the server calls `import_plugins()` in local server when the settings file is empty.